go/vt/vtctl/vtctl.go

```diff
@@ -3168,9 +3174,19 @@ func commandOnlineDDL(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag
 				condition, bindErr = sqlparser.ParseAndBind("migration_context=%a", sqltypes.StringBindVariable(arg))
 			}
 		}
+		order := " order by added_timestamp ascending"
+		if *orderBy == "descending" {
+			order = " order by added_timestamp descending"
```

Comment:
This column is not indexed, which means this will be a full table scan. Suggestion: `order by id desc`. `id` is the `PRIMARY KEY` for this table, and is an `AUTO_INCREMENT`. Thus, both `id` and `added_timestamp` are monotonously increasing.